Sources/AckGen/Acknowledgement.swift

```diff
@@ -17,12 +17,16 @@ public struct Acknowledgement: Codable {
         self.license = license
     }
 
-
-    /// <#Description#>
-    /// - Parameter plistName: the property list's filename without extension
-    /// - Returns: Array of objects containing title and
-    public static func all(fromPlist plistName: String = "Acknowledgements") -> [Acknowledgement] {
-        guard let path = Bundle.main.path(forResource: plistName, ofType: "plist"),
+    /// Fetch all the acknowledgements from the acknowledgement property list's file.
+    /// - Parameters:
+    ///   - plistName: the property list's filename without extension. Default to `Acknowledgements`.
```

Comment:
[super mega tiny nit]

```suggestion
    ///   - plistName: the property list's filename without extension. Defaults to `Acknowledgements`.
```